app/views/assessor_interface/application_forms/index.html.erb

```diff
@@ -63,6 +63,12 @@
         <% end %>
       <% end %>
     </section>
+
+    <section id="app-applications-show-all-applicants">
+      <%= f.govuk_check_boxes_fieldset :display, legend: { size: "s" }, small: true do %>
+        <%= f.govuk_check_box :display, :show_all, label: { text: "All applications" }, checked: @view_object.filter_form.display&.include?("show_all") %>
```

Comment:
```suggestion
        <%= f.govuk_check_box :display, :show_all, label: { text: "Show applications completed older than 90 days ago" }, checked: @view_object.filter_form.display&.include?("show_all") %>
```

Reply:
Makes sense - I think I'll need to adapt this in the spec too